### PR TITLE
Done

### PR DESCRIPTION
**Pull request by bc86**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,6 +23,15 @@
...
 
+          <div class="student-card" id="brett-cole-card">
+            <a href="students/brett-cole.html">
+              <div class="view-profile-div">
+                <h3 class="view-profile-text">Check Me Out</h3>
+              </div>
+              <div class="card-text-container">
+                <h4 class="student-name">Brett Cole</h4>
+                <p class="student-location">Newark, DE</p>
+
```
Comment (bhollan) [inline index.html]: Hi!  After reviewing the changes, it looks like you're missing two closing `</div>` tags here.

Comment (bc86) [inline index.html]: I just got your message sorry about that.  I didn't mean to.  What would be the best way to fix this so not to mess it up more?  and you said I am missing closing divs?

**Comment by bhollan** [comment]
Hi @bc86 !  It also looks like you overwrote another student's profile file to create yours.  If you could reinstate that, I think we're ready to roll!

**Comment by bc86** [comment]
i believe its all set know please let me know if i need to change anything else. The overwrite was a complete accident

**Comment by bhollan** [comment]
Hi @bc86!

It looks like we're getting there!  It looks like you've made your file as a new file, but you haven't re-instated `aaron-enser.html` yet.  Do let us know if you need any help figuring it out!

CC: @aturkewi

**Comment by ivalentine** [comment]
Hi @bc86! Following up on Brian's request above — if you need any help making these changes, just let us know.

cc @aturkewi

**Comment by ivalentine** [comment]
@bc86 I'm going to close this PR for now — if you'd like to make those changes, feel free to re-open it. Thanks!

**Pull request closed without merging**